apps/web/src/data/ecosystem.json

```
@@ -4685,7 +4685,7 @@
     "description": "WLTH opens the door to tokenised pre-IPO opportunities like xAI and SpaceX, while AI-powered strategies help your assets grow on autopilot. One app. No barriers.",
     "imageUrl": "/images/partners/wlth.webp",
     "category": "Consumer",
```

You may want to lowercase this as well while you're at it so that it is consistent with other listings in the file.

```suggestion
    "category": "consumer",
```